docs/sources/visualizations/histogram-draft.md

```diff
@@ -0,0 +1,107 @@
+
+If the approximated value is larger than the largest bucket (excluding the `+Inf` bucket), Prometheus will give up and give you the value of the largest bucket’s `le` back.
+
+We can then make our approximation of the third quartile with the following query:
+
+```bash
+histogram_quantile(0.75, uploaded_image_bytes_bucket)
+```
+
+**Note:** When talking about service level, the precision of quantile estimations is relevant. Historically, a lot of services are defined as something like “the p95 latency may not exceed 0.25 seconds.” Assuming we have a bucket for `le=0.25`, we *can* accurately answer whether or not the p95 latency does exceed 0.25 or not.
+
+However, since the p95 value is approximated, we cannot tell definitively if p95 is, say, 0.22 or 0.24 without a bucket in between the two. A way of phrasing this requirement so we do get an accurate number of how close we are to violating our service level is “the proportion of requests in which latency exceeds 0.25 seconds must be less than 5 percent.” Instead of approximating the p95 and seeing if it’s below or above 0.25 seconds, we precisely define the percentage of requests exceeding 0.25 seconds using the methods from above.
+
+## Buckets' distribution
+
+When users create a **bar gauge** panel to visualize `uploaded_image_bytes_bucket` and set the label to `{{le}}`, they will notice the following:
+
+- The values may be inaccurate because *Mean* is the default calculation the bar gauge performs on the data it receives. Under the **Panel > Display > Value** option, change this to *Last* for the correct value.
+- The buckets are out of order because they are being ordered alphabetically rather than numerically (i.e. 10 is smaller than 2, since 1 is smaller than 2).
+- The cumulative nature of the histogram, as every bucket contains more elements than the previous.
+
+The option to change the **format** of the Prometheus data from `Time series` to `Heatmap` allows users to tell Grafana it is working with a histogram </em>and</em> they would like to sort the buckets to only show distinctive counts for each bucket.
+
+## Buckets' distribution over time
+
+Since a bar gauge does not contain any temporal data, users must use a **Heatmap** to see the same visualization over time. After switching the panel type to **Heatmap**, the following adjustments are required to display it properly:
+
+- Change the **Panel > Axes > Data Format > Format** option from *Time series* to *Time series buckets* as buckets are pre-defined.
+- Slightly change the query to show the increase per histogram block rather than the total count of the bucket. The new query is `sum(increase(image_uploaded_bytes_bucket[$__interval])) by (le)`. When users have a temporal dimension in their visualization, they must make sure the query takes advantage of that.
```

Comment:
```suggestion
- Update the query to show the increase per histogram block rather than the total count of the bucket. The new query is `sum(increase(image_uploaded_bytes_bucket[$__interval])) by (le)`. This ensures that any temporal dimension in the visualization is incorporated into the query.
```